src/python/grpcio_tests/tests/testing/_server_application.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An example gRPC Python-using server-side application."""
+
+import grpc
+
+# requests_pb2 is a semantic dependency of this module.
+from tests.testing.proto import requests_pb2  # pylint: disable=unused-import
+from tests.testing.proto import services_pb2
+from tests.testing.proto import services_pb2_grpc
+
+from tests.testing import _application_common
+
+
+class FirstServiceServicer(services_pb2_grpc.FirstServiceServicer):
```

Comment:
It'd be a little textual convenience and a radical departure from the way the system under test is expected to be used in non-test production circumstances.

While this example illustrates a servicer implementation that takes no construction parameters, real servicer implementations are expected to take application-determined, application-semantic construction parameters at least some of the time. Presumably the testing framework wouldn't be able to infer how to instantiate the servicer class then?